lib/http-bridge/test/integration/Cardano/Wallet/HttpBridge/NetworkSpec.hs

```diff
@@ -55,9 +57,9 @@ spec = do
     describe "Happy paths" $ around withBridge $ do
         it "get unstable blocks for the unstable epoch" $ \(bridge, _) -> do
             let action = runExceptT $ do
-                    (SlotId ep sl) <- slotId <$> networkTip' bridge
+                    (SlotId (EpochNo ep) (SlotNo sl)) <- slotId <$> networkTip' bridge
                     let sl' = if sl > 2 then sl - 2 else 0
-                    blocks <- nextBlocks' bridge (mkCursor bridge $ SlotId ep sl')
+                    blocks <- nextBlocks' bridge (mkCursor bridge $ SlotId (EpochNo ep) (SlotNo sl'))
```

Comment:
After the renaming, this line is now over the line length limit. :upside_down_face: 

Recommend
```suggestion
                    blocks <- nextBlocks'
                        bridge
                        (mkCursor bridge $ SlotId (EpochNo ep) (SlotNo sl'))
```